Detectors/TPC/qc/include/TPCQC/ExpertVis.h

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   ExpertVis.h
+/// @author Berkin Ulukutlu, berkin.ulukutlu@cern.ch
+///
+
+#ifndef AliceO2_TPC_QC_ExpertVis_H
+#define AliceO2_TPC_QC_ExpertVis_H
+
+#include <vector>
+
+// root includes
+#include "THnSparse.h"
+#include "TH1F.h"
+#include "TH2F.h"
```

Comment:
Not used I think.
```suggestion
```

Reply:
If the TH2F is not included I get an error from the helpers.h. I keep it for now
```
/home/berki/alice/sw/SOURCES/O2/ExpertVis/0/Detectors/TPC/qc/include/TPCQC/Helpers.h:48:26: error: 'TH2' was not declared in this scope; did you mean 'TH2F'?
48 | void setStyleHistogram2D(TH2& histo);
```

Reply:
Can you try to add missing forward declarations
```
class TH1;
class TH2;
```
in `TPCQC/Helpers.h` and see if this solves the issue.

Reply:
If you need to keep it it shouldn't be in the header, by the way. You don't use the Helpers in the header...

Reply:
ok yes adding these to the Helpers.h worked